build/upgrade_pdfjs.sh

```diff
@@ -5,10 +5,9 @@
 #this fix or fixes it on it's own
 
 
-DOC_COMPILER_BINARY="build/closure-compiler-v20200719.jar"
-DOC_STATIC_ASSETS_BRANCH=${1:-master}
-DOC_STATIC_ASSETS_VERSION=$(./build/current_version.sh)
-DOC_STATIC_ASSETS_PATH="src/third-party/doc/${DOC_STATIC_ASSETS_VERSION}"
+DOC_STATIC_ASSETS_PDFJS_VERSION=${1}
+DOC_STATIC_ASSETS_PATH="src/third-party/doc/${DOC_STATIC_ASSETS_PDFJS_VERSION}"
+if  [[ ${1:0:1} -ge "4" ]]; then JS_FILE_EXTENSION="mjs"; else JS_FILE_EXTENSION="js"; fi
```

Comment:
I don't think we need to support `.js` anymore. It's probably safe to assume the use of `mjs` going forward.